Documentation/Fr/Index.xyl

```diff
@@ -208,14 +205,45 @@ echo $title->toAscii();
  * Will output:
  *     Un ete brulant sur la cote
  */</code></pre>
+  <p>Nous pouvons aussi transformer de l'arabe ou du japonais vers de l'ASCII.
+  Les symboles, comme les symboles Mathématique ou les emojis, sont aussi
```

Comment:
An “s” is missing but I keep the uppercase because Mathematics is a science, and science needs an uppercase ;-).